docs/src/LinearAlgebra/vinberg.md

```diff
@@ -0,0 +1,53 @@
+```@meta
+CurrentModule = Oscar
+DocTestSetup = Oscar.doctestsetup()
+```
+
+# Vinberg's algorithm
+
+ A Lorentzian lattice $L$ is an integral $\Z$ - lattice in hyperbolic space.
+ $L$ is called reflective if the set of fundamental roots $\~{R}(L)$ is finite.
+
+ See for example [Tur17](@cite) for the theory of Arithmetic Reflection Groups and Reflective Lorentzian Lattices.
+
+## Description 
+ The algorithm constructs a fundamental polyhedron $P$ for a Lorentzian lattice $L$ by computing its fundamental roots $r$.
+
+ Choose $v_0$ with $v_0^2 > 0$ as a point that $P$ should contain.
+
+ Let $Q$ be the corresponding gram matrix of $L$. A fundamental root $r$ satisfies
+ - the vector $r$ is primitive
+ - reflection by $r$ preserves the lattice, i.e. $\frac{2}{r^2}*r*Q$ is an integer matrix.
```

Comment:
This bullet point ends with a period, the others don't. perhaps end all of them with a comma or semicolon, except for the last one (which should end with a period)?